### PR TITLE
Update 0028-GOVE-governance.md 0028-GOVE-077

### PR DESCRIPTION
**Pull request by vega-paul**
Rewording of the AC 0028-GOVE-077

### CODE REVIEW

File: protocol/0028-GOVE-governance.md [hunk 1]
```diff
@@ -529,7 +529,7 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
 
 - Proposals are either permitted or rejected according to the following source/destination combinations. (<a name="0028-GOVE-075" href="#0028-GOVE-075">0028-GOVE-075</a>)
 - Transfer amount and fraction of balance are mutually exclusive and will cause the proposal to reject (<a name="0028-GOVE-076" href="#0028-GOVE-076">0028-GOVE-076</a>)
-- Invalid source and destination account types will cause the proposal to reject (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
+- Invalid source and destination account types will not be included in the transaction and will result in a wallet rejection (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
```
Comment (davidsiska-vega) [inline protocol/0028-GOVE-governance.md]: Wallet rejection is not precise enough; people may choose to not use the vega wallet to submit transactions. It must be rejected by the chain.

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]

File: protocol/0028-GOVE-governance.md (hunk 1 above)
Comment (vega-paul) [reply protocol/0028-GOVE-governance.md]: updated - please check @davidsiska-vega

File: protocol/0028-GOVE-governance.md [hunk 2]
```diff
@@ -529,11 +529,10 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
 
 - Proposals are either permitted or rejected according to the following source/destination combinations. (<a name="0028-GOVE-075" href="#0028-GOVE-075">0028-GOVE-075</a>)
 - Transfer amount and fraction of balance are mutually exclusive and will cause the proposal to reject (<a name="0028-GOVE-076" href="#0028-GOVE-076">0028-GOVE-076</a>)
-- Invalid source and destination account types will cause the proposal to reject (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
+- Invalid source and destination account types will not be included in the transaction and will cause the transaction to be rejected (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
```
Comment (gordsport) [inline protocol/0028-GOVE-governance.md]:
```suggestion
- Governance transfer proposals with invalid source or destination account types rejected by the blockchain (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
```

Comment (gordsport) [reply protocol/0028-GOVE-governance.md]: @vega-paul - can you please also add the wallet case to the CLIE spec?

File: protocol/0028-GOVE-governance.md [hunk 3]
```diff
@@ -529,11 +529,10 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
...
 - Source Type can be any of the predefined types in the above table (<a name="0028-GOVE-078" href="#0028-GOVE-078">0028-GOVE-078</a>)
 - Source can be left blank for a transfer type of Network Treasury (<a name="0028-GOVE-079" href="#0028-GOVE-079">0028-GOVE-079</a>)
-- Source can be left blank for a transfer type of Network Insurance Pool (<a name="0028-GOVE-080" href="#0028-GOVE-080">0028-GOVE-080</a>)
-- For transfer source types of Market Insurance the source must be a valid market ID  else is rejected (<a name="0028-GOVE-081" href="#0028-GOVE-081">0028-GOVE-081</a>)
+- For proposal source/destination types of Market Insurance the source/destination must be a valid market ID else is proposal is rejected (<a name="0028-GOVE-081" href="#0028-GOVE-081">0028-GOVE-081</a>)
```
Comment (gordsport) [inline protocol/0028-GOVE-governance.md]:
```suggestion
- For proposal source/destination types of Market Insurance the source/destination must be a valid `marketID` else the proposal is rejected (<a name="0028-GOVE-081" href="#0028-GOVE-081">0028-GOVE-081</a>)
```

**Review by gordsport — CHANGES_REQUESTED** [review]

File: protocol/0028-GOVE-governance.md [hunk 4]
```diff
@@ -529,11 +529,10 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
 
 - Proposals are either permitted or rejected according to the following source/destination combinations. (<a name="0028-GOVE-075" href="#0028-GOVE-075">0028-GOVE-075</a>)
 - Transfer amount and fraction of balance are mutually exclusive and will cause the proposal to reject (<a name="0028-GOVE-076" href="#0028-GOVE-076">0028-GOVE-076</a>)
-- Invalid source and destination account types will cause the proposal to reject (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
+- Governance transfer proposals with invalid source or destination account types rejected by the blockchain (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
```
Comment (davidsiska-vega) [inline protocol/0028-GOVE-governance.md]:
```suggestion
- Governance transfer proposals with invalid source or destination account types will get rejected by the blockchain. (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
```

File: protocol/0028-GOVE-governance.md [hunk 5]
```diff
@@ -529,11 +529,10 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
...
 - Source Type can be any of the predefined types in the above table (<a name="0028-GOVE-078" href="#0028-GOVE-078">0028-GOVE-078</a>)
 - Source can be left blank for a transfer type of Network Treasury (<a name="0028-GOVE-079" href="#0028-GOVE-079">0028-GOVE-079</a>)
-- Source can be left blank for a transfer type of Network Insurance Pool (<a name="0028-GOVE-080" href="#0028-GOVE-080">0028-GOVE-080</a>)
-- For transfer source types of Market Insurance the source must be a valid market ID  else is rejected (<a name="0028-GOVE-081" href="#0028-GOVE-081">0028-GOVE-081</a>)
+- For proposal source/destination types of Market Insurance the source/destination must be a valid `marketID` else the proposal is rejected (<a name="0028-GOVE-081" href="#0028-GOVE-081">0028-GOVE-081</a>)
```
Comment (davidsiska-vega) [inline protocol/0028-GOVE-governance.md]:
```suggestion
- For proposal source/destination types of Market Insurance the source/destination must be a valid `marketID` else the proposal is rejected by the blockchain. (<a name="0028-GOVE-081" href="#0028-GOVE-081">0028-GOVE-081</a>)
```

**Pull request closed without merging**